src/modules/setavatar.js

```diff
@@ -9,20 +10,20 @@ const VALIDATE_IMG = /^http(?:s):\/\/([\w@:%.+~#=-]{1,256}\.[a-zA-Z0-9()]{1,6})\
 module.exports = bot => {
   bot.registerCommand("setavatar", async (msg, args) => {
     let url = args[0];
-    if (! VALIDATE_IMG.test(url)) return bot.createMessage(msg.channel.id, "<:dynoError:696561633425621078> Invalid image URL");
+    if (! VALIDATE_IMG.test(url)) return utils.sendError(msg, "Invalid image URL");
     let response;
     try {
       response = await superagent.get(url).accept("image/*").responseType("arraybuffer");
     } catch (error) {
       if (error.response) { // Server responded with error
-        return bot.createMessage(msg.channel.id, `<:dynoError:696561633425621078> Server responded with: ${error.status} ${error.message}\n${error.response.text || ""}`);
+        return utils.sendError(msg, `Server responded with: ${error.status} ${error.message}\n${error.response.text || ""}`);
       } // Something happened with request
-      return bot.createMessage(msg.channel.id, "<:dynoError:696561633425621078> Unable to send request: " + error.message);
+      return utils.sendError(msg, "Unable to send request: " + error.message);
```

Comment:
Is it worth sending the stack trace to the log channel here?

Reply:
It doesn't? It's the same exact error